inductiva/fluids/dam_break/dam_break.py

```diff
@@ -53,7 +53,7 @@ def simulate(
         self,
         simulator: Simulator = DualSPHysics(),
         machine_group: Optional[resources.MachineGroup] = None,
-        device: Literal["cpu", "gpu"] = "gpu",
+        device: Literal["cpu", "gpu"] = "cpu",
```

Comment:
ok! I suspect that we are having problems with the GPU, right?
Should there be an issue?

Reply:
We are moving to fix this, but there should be some introspection on the backend to understand if the requests we are asking here are being fulfilled!